airbyte-webapp/src/core/resources/DestinationDefinition.ts

```diff
@@ -31,6 +33,37 @@ export default class DestinationDefinitionResource
   > {
     return {
       ...super.listShape(),
+      fetch: async (
+        params: Readonly<Record<string, string | number>>
+      ): Promise<{ destinationDefinitions: DestinationDefinition[] }> => {
+        const definition = await this.fetch(
+          "post",
+          `${this.url(params)}/list`,
+          params
+        );
+        const latestDefinition = await this.fetch(
```

Comment:
does this mean every time you call /list you will also call /list_latest? this is okay, but if you are doing this, we might implement more aggressive caching for list_latest because it's pulling data from an external webserver.

Reply:
heads up @davinchia

Reply:
We call it every time on initial webapp opening as there is no other way to detect if there are some outdated containers or not from UI. Ideally, this data could be received on backend and UI can just get this as sort of notifications list or flag in settings or preferences call.

Reply:
cool. okay if it's just when initially opening it then, we probably don't need caching now.

Reply:
thanks for the heads up!

Reply:
to make sure I understand, what does 'initial webapp opening' mean? everytime the page is refreshed?

Reply:
If saying `page is refreshed` you mean refresh button in a browser - then yes.